### ISSUE TITLE
crosswords: No compatibility with Dragon NaturallySpeaking (accessibility)

### ISSUE THREAD

**Issue by ScottPainterGNM** [guardian/frontend#10349]
Feedback from a disabled gentleman that uses the above to help him do the crosswords, this works in R2 but new Crosswords aren't compatible.

**Comment by OliverJAsh** [comment]
The old R2 crossword grid was rendered as structured HTML, so the screenreader can pick up the clue for each cell. Unfortunately the new crossword grid is rendered as an SVG, which of course loses all the structure. Maybe we should have built it as HTML :-( /cc @robertberry 

On a related note, it makes me sad that we need an "accessible version"—shouldn't the main crossword page be the accessible version?

For now, will the accessible version do the job for screenreader users like this one—who do not have visual impairments and are used to using the R2 "inaccessible version" grid?

**Comment by gtrufitt** [comment]
There is also an issue with using the :before pseudo element as it is not necessarily read by a screen reader - http://www.w3.org/TR/WCAG10-CSS-TECHS/#Generated 

It may be worth testing to see if we can apply better screen reader accessibility on top of keyboard support by using the aria-labelledby attribute to read out current clue when navigating the crossword with a keyboard?

**Comment by OliverJAsh** [comment]
Good thinking @gtrufitt. The trouble is we re-use the same `input` field for the whole crossword. I'm not sure if updating it programatically (when you navigate to another clue) would cause the screenreader to read it out again.

**Comment by gtrufitt** [comment]
> I'm not sure if updating it programatically (when you navigate to another clue) would cause the screenreader to read it out again.

That was my thoughts too, I guess we would need to test. I actually realised that naturally speaking is a voice control tool, not a screen reader, so we may have a different issue here - I don't know how the voice control works so this might be something we need to investigate further too.

**Comment by ScottPainterGNM** [comment]
I'm going to close this one off for now unless we have a chance to revisit this.
